scripts/aws/cfn/storage.yml

```diff
@@ -69,6 +69,13 @@ Resources:
       KeySchema:
         - AttributeName: "aarQRCodeValue"
           KeyType: "HASH"
+      GlobalSecondaryIndexes:
+        - IndexName: "iun-index"
+          KeySchema:
+            - AttributeName: "iun"
+              KeyType: "HASH"
+          Projection:
+            ProjectionType: "ALL"              
       BillingMode: "PAY_PER_REQUEST"
```

Comment:
Spero non si scassi il CloudFormation aggiungendo il GSI... in caso sono dolori... chiedi a Marco o segui l'aggiornamento dello stack

Reply:
ho letto la doc https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-dynamodb-table.html chiedo Marco per conferma

Reply:
@masciale89 @antoniopezzuto chiedete a @fdammassa il deploy sull'ambiente di prova devops così scopriamo cosa succede prima di fare la merge in dev

Reply:
@antoniopezzuto ho fatto un commit perché mancava la definizione dell'attributo in `AttributeDefinitions` . Il deploy su hotifx ha aggiornato correttamente la tabella aggiungendo il GSI.
/cc @masciale89 @marco-vit-pagopa